### ISSUE TITLE
[Bug]: In file dialog Navigation to project home dir does not work.

### ISSUE THREAD

**Issue by Oen1997** [apache/hop#3899]
### Apache Hop version?

2.8

### Java version?

Java(TM) SE Runtime Environment (build 1.8.0_411-b09)

### Operating system

Windows

### What happened?

Clicking on button P does nothing.
![image](https://github.com/apache/hop/assets/210065/7d149bca-61e2-4a58-9409-4929c5565c73)

### Issue Priority

Priority: 3

### Issue Component

Component: Hop Gui

**Comment by nadment** [comment]
Could you try with Java 11

**Comment by Oen1997** [comment]
Changed java to:
openjdk version "11.0.23" 2024-04-16 LTS
Issue still exists.

_Few more notes_: 
It does not work from any file related transform, when I press "Browse" . 
It works if I open File>Save, or File>Open dialog.
When I try to open projects folder in this case, I get error:
![image](https://github.com/apache/hop/assets/210065/e5e29b24-6e21-4126-8a5c-7fa8ec5f6a08)

And reason is: 
![image](https://github.com/apache/hop/assets/210065/271a2119-75e5-4c29-aaae-3066e053c20c)

**Comment by hansva** [comment]
could you add your project configuration too?
Just to confirm, your PROJECT_HOME is ${HOP_CONFIG_FOLDER}/projects/PartMasterData ?

I will guess that we don't support nested variables causing path expansion to go wrong somewhere

**Comment by Oen1997** [comment]
Yes, thats correct, different Config folders on my Pc, and Hop server, so I must use such approach:
![image](https://github.com/apache/hop/assets/210065/cccfdc24-7cd5-4444-90a7-207c563361f9)

Solution might be to resolve variables in those dialogs, instead of trying to use them in place.

**Comment by Oen1997** [comment]
By the way, tested single variable: 
![image](https://github.com/apache/hop/assets/210065/7cd74f74-c1ee-4181-8887-1cdf48b46063)

**Comment by hansva** [comment]
Guess that will be the root cause of the issue.

**Comment by hansva** [comment]
related to #3324
